### PR TITLE
[W5.2e][T11-A2][Siddharth Madhavan]

### PR DESCRIPTION
**Pull request by sidhmads**
Added isAnyNull test case to UtilsTest.java

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +38,12 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
```
Comment (OuyangDanwen) [inline test/java/seedu/addressbook/common/UtilsTest.java]: This is not the correct way of testing this method. You should use assertTrue, assertFalse to test the output when you are using the method isAnyNull(). Refer to one of your classmates' PR if you are not sure.

**Comment by jshoung** [comment]
Other than mentioned errors, overall good enhancements and did more than one.

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 1 above)
Comment (sidhmads) [reply test/java/seedu/addressbook/common/UtilsTest.java]: I have no clue why am i failing the CI test

Comment (sidhmads) [reply test/java/seedu/addressbook/common/UtilsTest.java]: pls help

Comment (sidhmads) [reply test/java/seedu/addressbook/common/UtilsTest.java]: The test cases passed on my terminal however it is failing in Travis CI. Not sure why

Comment (OuyangDanwen) [reply test/java/seedu/addressbook/common/UtilsTest.java]: Check the assertEqual() on line 48 of FindCommandTest.java? It passed on your computer?

Comment (OuyangDanwen) [reply test/java/seedu/addressbook/common/UtilsTest.java]: It seems that the two lists that you are comparing have different contents.

Comment (sidhmads) [reply test/java/seedu/addressbook/common/UtilsTest.java]: The assertEqual on line 48, passes on my computer

Comment (OuyangDanwen) [reply test/java/seedu/addressbook/common/UtilsTest.java]: In that case, just send me a screenshot on Slack first. I will give you the mark. Don't worry about it.

**Pull request closed without merging**